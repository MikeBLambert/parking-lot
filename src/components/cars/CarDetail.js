import React from 'react'

export default function CarDetail() {
  return (
    <div>
      
    </div>
  )
}
