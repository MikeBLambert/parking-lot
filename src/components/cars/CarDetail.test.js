import React from 'react';
import { shallow } from 'enzyme';
import CarDetail from './CarDetail';

describe('CarDetail component', () => {
  it('renders a CarDetail component', () => {
    const wrapper = shallow(<CarDetail />);
    expect(wrapper).toMatchSnapshot();

  });
});
