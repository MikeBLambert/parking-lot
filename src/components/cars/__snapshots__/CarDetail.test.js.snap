// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarDetail component renders a CarDetail component 1`] = `<div />`;
